addons/project/project.py

```diff
@@ -1150,10 +1150,11 @@ def message_new(self, cr, uid, msg, custom_values=None, context=None):
         defaults = {
             'name': msg.get('subject'),
             'planned_hours': 0.0,
+            'partner_id': msg.get('author_id', False)
         }
         defaults.update(custom_values)
         res = super(task, self).message_new(cr, uid, msg, custom_values=defaults, context=context)
-        email_list = tools.email_split(msg.get('to', '') + ',' + msg.get('cc', ''))
+        email_list = tools.email_split((msg.get('to') or '') + ',' + (msg.get('cc') or ''))
```

Comment:
@tde-banana-odoo  this was the fixed for "TypeError: coercing to Unicode: need string or buffer, NoneType found" here what happen sometime, the keys('cc' or 'to') are available in dict but the value they hold is None, so that default value set in get method(msg.get('cc', '')) will never execute and it then append none value to string, which will lead to traceback.